views/main.view.js

```diff
@@ -1026,33 +1026,43 @@ class View {
                     let storyTooltipText = "";
                     let lastInBranch = false;
                     const name = action.name.toLowerCase().replace(/ /gu, "_");
-                    const storyAmt = _txt(`actions>${name}`, "fallback").split("⮀").length - 1;
-                    let storiesUnlocked = 0;
-                    for (let i = 1; i <= storyAmt; i++) {
-                        storyTooltipText += "<p>"
-                        const storyText = _txt(`actions>${name}>story_${i}`, "fallback").split("⮀");
-                        if (action.storyReqs(i)) {
-                            storyTooltipText += storyText[0] + storyText[1];
-                            lastInBranch = false;
-                            storiesUnlocked++;
-                            if (action.visible() && action.unlocked() && completedActions.includes(action.varName)) {
-                                actionLog.addActionStory(action, i, init);
+
+                    const rawStoriesDataForAction = _txtsObj(`actions>${name}`, "fallback")[0].children;
```

Comment:
The `Action` class, since it extends `Localizable`, has a `.txtsObj` property which will give you the proper XML object for that action's localizations. You can use that instead of calling `_txtsObj`, but I think I'd prefer if you take it a step further; add a method to `Action` called, say, `getStoryTexts()`, that retrieves an array consisting of all applicable stories, and which can take care of all the `.split("⮀")` nonsense on its own. (should be a method, rather than a property accessor, since one of the changes I'll eventually be making is that a single `Action` can be valid/present in all zones, but may have different stories depending on which it's performed in.)